src/build.py

```diff
@@ -1397,7 +1406,7 @@ def TestBare():
       wasmjs=os.path.join(INSTALL_LIB, 'wasm.js'),
       extra_files=[os.path.join(INSTALL_LIB, 'musl.wasm')])
 
-  if IsMac():
+  if IsMac() and 'JSC' not in buildbot.WarnedList():
```

Comment:
Yeah, I like that better. I guess that might go along with keeping `Fail()` as-is, and `DidStepFail` doesn't care whether it warned or failed.